lib/board.rb

```diff
@@ -0,0 +1,45 @@
+class Board
```

Comment:
Great job using the most expressive methods available (e.g. `count`, `between`, `all`)!